flight/mcu/lib/teensyShared/dataLog.cpp

```diff
@@ -133,10 +141,9 @@ int32_t flush_iterator = 0;
 void logData(File* dataFile, sensorDataStruct_t* data) {
     // Write raw bytes to SD card.
     dataFile->write((const uint8_t*)data, sizeof(*data));
-
     // Flush data once for every 1000 writes (this keeps the ring buffer in sync
     // with data collection)
-    if (flush_iterator == 1000) {
+    if (flush_iterator == 50) {
```

Comment:
1. Update the comment above. 2. That's a really drastic change for the flush iterator! What difference does this make? I may also not be too clear on what exactly the "flush" does.

Reply:
flush is the step that actually writes the data to the sd card. A flush every 1000 iters is only once per 5-10 seconds so more potential for data loss.